### DISCUSSION TITLE
Inquiry Regarding Sudden Drop in Reputation

### DISCUSSION THREAD

**Discussion by vishxly** [dailydotdev/daily#1485] [General]
Dear Support Team,

I hope this message finds you well. My username is @vyan, and I recently noticed that my reputation, which was around 1.5k, has dropped to 0. I am unsure whether this is due to a bug or another issue. Could you please provide me with details on why this happened?

I would greatly appreciate any clarification or assistance you can offer in addressing this matter.

Thank you for your support!

Best regards,  
Vishal Yadav  
@vyan

**Comment by idoshamun** [comment]
Hi @vishxly,

I replied to email you sent

**Reply by vishxly** [reply]
Thanks for addressing the issue!
